config/settings.yml

```diff
@@ -30,9 +30,12 @@ CAP:
 PUBMED:
   LOG: log/pubmed.log
   FETCH_PATH: /entrez/eutils/efetch.fcgi?db=pubmed&retmode=xml
+  SEARCH_PATH: /entrez/eutils/esearch.fcgi?db=pubmed&retmode=xml&retmax=100000
   API_KEY: a_fake_pubmed_key
   BASE_URI: https://eutils.ncbi.nlm.nih.gov
   ARTICLE_BASE_URI: https://www.ncbi.nlm.nih.gov/pubmed/
+  # In number of days
+  regular_harvest_timeframe: 21
```

Comment:
Can we set this value to 'nil' or something else to eliminate the time filtering constraint?  This would allow us to mimic the behavior of the WoS harvesting, which does a full harvest for all time for new or updated users.  

e.g. from settings.yml, these are the WoS values that used in the `lib/cap/authors_poller.rb` class that we will need to add to

 ``` 
update_timeframe: null # default WoS symbolicTimeSpan for updated authors resulting from nightly cap polling
new_author_timeframe: null # default WoS symbolicTimeSpan for new authors resulting from nightly cap polling (null == for all time)
```

Reply:
This setting is currently only used in `harvest_authors_update` rake task, so it won't act as a default elsewhere.

Reply:
I added pubmed to `lib/cap/authors_poller.rb` with suggested defaults.